src/openzaak/components/besluiten/tests/test_besluit_with_external_zaak.py

```diff
@@ -169,7 +172,10 @@ def test_update_external_zaak(self):
 
         besluit.refresh_from_db()
 
-        self.assertEqual(besluit._previous_zaak_url, zaak_old)
+        self.assertEqual(besluit._previous_zaak_base_url_id, self.zaken_service.id)
+        self.assertEqual(
+            besluit._previous_zaak_relative_url, zaak_old[len(self.base) :]
+        )
```

Comment:
the implementation detail doesn't really matter - I think the assert should remain the same, it was already leaky abstraction enough.

Reply:
The assertion is changed back